src/App.js

```diff
@@ -854,6 +854,7 @@ function FullApp() {
                   savedSlippageAmount={savedSlippageAmount}
                   setPendingTxns={setPendingTxns}
                   connectWallet={connectWallet}
+                  savedShouldDisableOrderValidation={savedShouldDisableOrderValidation}
```

Comment:
About naming - should the GlpSwap component know that this prop is "saved" somewhere? Maybe just shouldDisableOrderValidation?

Reply:
The value is prefixed with `saved` because it's saved in `localStorage`. This is a common patter that has been used throughout the whole app/